docker-compose.yml

```diff
@@ -50,10 +50,31 @@ services:
       context: ./dff/utils/docker
       dockerfile: dockerfile_stats
     image: ghcr.io/deeppavlov/superset_df_dashboard:latest
```

Comment:
We should build a local image in tests instead of using a remote image.
(as mentioned [here](https://github.com/deeppavlov/dialog_flow_framework/pull/227#discussion_r1369481990))

Reply:
This can be addressed by passing the `--build` flag to the `docker-compose up` command

Reply:
Apparently, it was already building instead of pulling an image in previous commits:
![image](https://github.com/deeppavlov/dialog_flow_framework/assets/61429541/223e6320-d0e3-401b-84b8-9f86041a07f3)

I don't know why tests didn't fail due to authentication errors.
Maybe because context storages were also launched it gave enough time to whatever dashboard awaits for.